drivers/rs485_seds/work/arbiter/implem/default/C/src/arbiter.c

```diff
@@ -0,0 +1,59 @@
+/* Body file for function Arbiter
+ * Generated by TASTE (kazoo/templates/skeletons/c-body/function.tmplt)
+ * You can edit this file, it will not be overwritten
+
+    !! IMPORTANT                                                        !!
+    !! When you modify your design (interface view), you must update    !!
+    !! the procedures corresponding to the provided interfaces in this  !!
+    !! file. The up-to-date signatures can be found in the header file. !!
+*/
+#include "arbiter.h"
+
+#include <assert.h>
+
+#define OUTPUT_BUFFER_SIZE 1000
+
+__attribute__((section(".sdramMemorySection")))
+static asn1SccPioHwas pioHwas;
+
+void arbiter_startup(void)
+{
+   asn1SccPioHwasPinConfig pinConfig;
+   pinConfig.mPortConfig = asn1SccPioHwas_Port_pioHwas_Port_A;
+   pinConfig.mPinConfig = 2;
+   pinConfig.mDirectionConfig = asn1SccPioHwas_Direction_pioHwas_Direction_Output;
+   pinConfig.mControlConfig = asn1SccPioHwas_Control_pioHwas_Control_Pio;
+   arbiter_RI_PioHwas_InitPin_Pi(&pioHwas, &pinConfig);
+}
+
+bool arbiter_is_control_packet_received
+      (const uint8_t *message_data, size_t length)
+{
+   if (length == 13 && message_data[8] == 0xFF) {
```

Comment:
Why don't we check the entire packet. And if we only need one byte to check,then why do we send 13 bytes?

Reply:
We send entire packet because one single byte won't pass escaping and checking CRC. We check only payload, which is just one byte.

Reply:
The packet is sent here by Assembler and Assembler uses Escaper, just like UART driver.